spec.html

```diff
@@ -1691,32 +1691,26 @@ <h1>Number::remainder ( _n_, _d_ )</h1>
         <emu-clause id="sec-numeric-types-number-add" oldids="sec-applying-the-additive-operators-to-numbers">
           <h1>Number::add ( _x_, _y_ )</h1>
           <p>The abstract operation Number::add takes arguments _x_ (a Number) and _y_ (a Number). It performs addition, producing the sum of _x_ and _y_ as determined using the rules of IEEE 754-2019 binary double-precision arithmetic:</p>
-          <ul>
-            <li>
-              If either operand is *NaN*, the result is *NaN*.
-            </li>
-            <li>
-              The sum of two infinities of opposite sign is *NaN*.
-            </li>
-            <li>
-              The sum of two infinities of the same sign is the infinity of that sign.
-            </li>
-            <li>
-              The sum of an infinity and a finite value is equal to the infinite operand.
-            </li>
-            <li>
-              The sum of two negative zeroes is *-0*. The sum of two positive zeroes, or of two zeroes of opposite sign, is *+0*.
-            </li>
-            <li>
-              The sum of a zero and a nonzero finite value is equal to the nonzero operand.
-            </li>
-            <li>
-              The sum of two nonzero finite values of the same magnitude and opposite sign is *+0*.
-            </li>
-            <li>
-              In the remaining cases, where neither an infinity, nor a zero, nor *NaN* is involved, and the operands have the same sign or have different magnitudes, the sum is computed and rounded to the nearest representable value using IEEE 754-2019 roundTiesToEven mode. If the magnitude is too large to represent, the operation overflows and the result is then an infinity of appropriate sign. The ECMAScript language requires support of gradual underflow as defined by IEEE 754-2019.
-            </li>
-          </ul>
+          <emu-alg>
+            1. Let _nx_ be ? ToNumber(_x_).
+            1. Let _ny_ be ? ToNumber(_y_).
+            1. If _nx_ is *NaN* or _ny_ is *NaN*, return *NaN*.
+            1. If _nx_ is *+&infin;*, then
+              1. If _ny_ is *-&infin;*, return *NaN*; else return *+&infin;*.
+            1. If _nx_ is *-&infin;*, then
+              1. If _ny_ is *+&infin;*, return *NaN*; else return *-&infin;*.
+            1. Assert: _nx_ is finite
```

Comment:
We could move this assert down one and assert that they're both finite. That might be more useful for a reader.